crates/ironrdp-web/src/session.rs

```diff
@@ -889,7 +921,7 @@ fn build_config(
         platform: ironrdp::pdu::rdp::capability_sets::MajorPlatformType::UNSPECIFIED,
         no_server_pointer: false,
         autologon: false,
-        no_audio_playback: true,
+        no_audio_playback: !enable_audio,
```

Comment:
thought: For a follow up PR, `no_audio_playback` should be changed to `enable_audio`, as I believe its a better naming.

Reply:
Done in https://github.com/Devolutions/IronRDP/pull/907